spec/src/main/asciidoc/Connectors.adoc

```diff
@@ -4655,12 +4641,12 @@ services that enable transactional context to be propagated across
 systems. The services provided by a transaction manager are not visible
 directly to the application components.
 
-The connector architecture defines a
+The Jakarta Connector Architecture defines a
 transaction management contract between an application server and a
 resource adapter and its underlying resource manager. The transaction
 management contract has two parts, depending on the type of transaction:
 
-a JTA _javax.transaction.xa.XAResource_ based
+a Jakarta Transactions _jakarta.transaction.xa.XAResource_ based
```

Comment:
I think XA resource is a JDK api

Reply:
According to the docs
https://javaee.github.io/javaee-spec/javadocs/
I think it's part of Java EE

Reply:
It is SE https://docs.oracle.com/en/java/javase/11/docs/api/java.transaction.xa/javax/transaction/xa/package-summary.html